addons/analytic/views/analytic_view.xml

```diff
@@ -101,7 +101,7 @@
                                 <field name="currency_id" options="{'no_create': True}" groups="base.group_multi_currency"/>
                             </group>
                         </group>
-                        <notebook>
```

Comment:
Why creating an empty notebook? The notebook is overrided only once, so let's create it in the other view.